flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/JsonFunctionsITCase.java

```diff
@@ -230,4 +233,76 @@ private static TestSpec jsonValue() throws Exception {
                         42,
                         DataTypes.INT());
     }
+
+    private static List<TestSpec> isJson() {
```

Comment:
I originally had more test cases, but removed some in a separate commit now since in the other PR that was one of the review feedback points.

Reply:
at some point we will need to improve the test base before people complain about our large expression testing times